packages/lib/CalEventParser.ts

```diff
@@ -137,6 +137,10 @@ export const getUid = (calEvent: CalendarEvent): string => {
   return uid ?? translator.fromUUID(uuidv5(JSON.stringify(calEvent), uuidv5.URL));
 };
 
+const getSeatReferenceId = (calEvent: CalendarEvent): string => {
+  return calEvent.attendeeSeatId ? `seatReferenceUId=${calEvent.attendeeSeatId}` : "";
+};
+
```

Comment:
Pass the seat referenceUId to the email reschedule and cancel links incase the attendee does not have a cal account